postcss.config.js

```diff
@@ -113,18 +113,12 @@ module.exports = ({
 				stage: 2,
 				env,
 				features: {
-					"custom-properties": true,
-					"logical-properties-and-values": false,
-					clamp: true,
-					"color-functional-notation": true,
 					"dir-pseudo-class": { preserve: true },
-					"nesting-rules": { noIsPseudoSelector: true },
```

Comment:
A lot of these settings can be removed as the defaults in the latest version align with our configurations.